pvacseq/lib/binding_filter.py

```diff
@@ -7,8 +7,6 @@
 
 def main(args_input = sys.argv[1:]):
     parser = argparse.ArgumentParser('pvacseq binding_filter')
-    parser.add_argument('input', type=argparse.FileType('r'),
-                        help="Input list of variants")
     parser.add_argument('fof', type=argparse.FileType('r'),
```

Comment:
I agree this is a separate PR, but I lean toward removing the fof and using a list arg.